Source/Urho3D/UI/FontFace.h

```diff
@@ -101,7 +97,7 @@ class URHO3D_API FontFace : public RefCounted
     SharedPtr<Texture2D> LoadFaceTexture(const SharedPtr<Image>& image);
 
     /// Parent font.
-    Font* font_{};
+    Font* font_;
```

Comment:
`{}` initializes pointer to `nullptr` while lack of it leaves pointer uninitialized. Afaik we prefer initializing fields within class body when writing new code. This should be `Font* font_ = nullptr;` really.

Reply:
There is only one `FontFace` constructor, _it_ initializes `font_`.

Reply:
With `font_{}` inited twice

Initiating in class body not always possible. Will some of members be initialized in constructor, and other in class body?

Reply:
In Urho, I believe member initialization almost always happens in the constructor's head with the exception of those belonging to `struct`s.